src/contracts/core/RewardsCoordinator.sol

```diff
@@ -303,23 +303,35 @@ contract RewardsCoordinator is
         require(msg.sender == operator, "RewardsCoordinator.setOperatorAVSSplit: caller is not the operator");
         require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator.setOperatorAVSSplit: split must be <= 10000 bips");
 
-        uint32 activatedAt = uint32(block.timestamp) + activationDelay;
-        uint16 oldSplit = _getOperatorSplit(operatorAVSSplitBips[operator][avs]);
-        _setOperatorSplit(operatorAVSSplitBips[operator][avs], split, activatedAt);
+        (uint16 oldSplit, uint32 oldActivatedAt) = _getOperatorSplit(operatorAVSSplitBips[operator][avs]);
+        require(
+            block.timestamp > oldActivatedAt,
```

Comment:
Ensuring that it's strictly greater than so that there's no overlap at all between activation delay periods.